NuGet.config

```diff
@@ -4,7 +4,7 @@
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
   <packageSources>
-    <clear />
+    <!--<clear />-->
```

Comment:
Let me know what package(s) needs to be available in the feeds below and I'll make sure it's there. Then this change can go away.

Reply:
it needs Akade.IndexedSet 1.0.1 and Bogus 34.0.2. How would it go btw, if I update the package? Which I will do soon for .NET 8...